openapi-spec.json

```diff
@@ -102,17 +102,114 @@
     },
     "components": {
         "schemas": {
-            "User": {
+            "Profile": {
                 "type": "object",
                 "properties": {
-                    "firstName": {
+                    "name": {
+                        "type": "string",
+                        "required": true
+                    },
+                    "discordId": {
                         "type": "string"
                     },
-                    "lastName": {
+                    "homePageUrl": {
                         "type": "string"
                     },
-                    "age": {
-                        "type": "number",
+                    "activeTimeStart": {
+                        "Type": "int",
```

Comment:
The `type` property should be in lower-case. This issue is repeated in multiple places